README.md

```diff
@@ -16,6 +16,21 @@ Docs and guides
 - [Setting up Emacs](http://cfme-tests.readthedocs.org/guides/editors.html)
 - [PR Review Process](https://github.com/ManageIQ/integration_tests/wiki/PR-Process)
 
+--Ubuntu Evironment Notes:
```

Comment:
I don't think this is the right place for such a note about environment setup.

The getting started doc would be a better place.
https://github.com/ManageIQ/integration_tests/blob/master/docs/guides/collections.rst

When we discussed the failures you were seeing, weren't you also missing a required system package in Ubuntu 18.x?  I think you should update quickstart's system module to include the missing package.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/scripting/quickstart/system.py#L95

Note quickstart is actually supposed to handle either pycurl SSL library via setting PYCURL_SSL_LIBRARY in the environment.  I think you should do a bit more debug on your system itself to make updates to quickstart to better handle Ubuntu and gnutls for pycurl.